portfolio/src/main/webapp/style.css

```diff
@@ -7,7 +7,50 @@
 #greeting-container {
   margin-top: 20px;
 }
+.nav{
```

Comment:
[nit] throughout this file, leave a space between class name and the {, so `.nav {`